external/source/exploits/CVE-2022-26904/SuperProfileDLL.sln

```diff
@@ -0,0 +1,31 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31912.275
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "SuperProfileDLL", "SuperProfileDLL\SuperProfileDLL.vcxproj", "{E57F8F02-C36B-4ADC-BD40-F25F5F73BD18}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|x64 = Debug|x64
+		Debug|x86 = Debug|x86
+		Release|x64 = Release|x64
+		Release|x86 = Release|x86

```

Comment:
Do we need to build for 64 and 32 bit if we only support 64 bit?

Reply:
Umm good point I can remove that 👍

Reply:
Addressed in https://github.com/rapid7/metasploit-framework/pull/16382/commits/24342e764c249854f0f5bd70b3a2107f4f6e3521